### PR TITLE
fix: fixes container repeat landing page

### PR DESCRIPTION
**Pull request by blendaf**

### CODE REVIEW

**Comment by Falcasantos** [comment]
Should the add new deck icon be below? It looks weird when it's above/beside the text
Capture d’écran 2020-04-20 à 19 28 57

**Comment by Falcasantos** [comment]
Also maybe it might be more user-friendly if the icon itself is inserted inline in the text:
Capture d’écran 2020-04-20 à 19 29 41

File: components/browse-decks/WelcomeRepeat.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components'
 
 const WelcomeContainer = styled.div`
-  display: inline;
```
Comment (juliuscc) [inline components/browse-decks/WelcomeRepeat.js:4]: I think this placed the text _under_ the 'create deck' button. Was this on purpose?

**Comment by juliuscc** [comment]
It might make sense to add a large button after the text that says 'Create your first deck' (or something similar). Also, the search stuff is not necessary.

If the full view is just the text, and then a big call to action, it could look less 'weird'.

**Comment by juliuscc** [comment]
This is where `Welcome` is used:

https://github.com/juliuscc/speak.exchange/blob/1d1ce952d8a2cc09bcb983fd21c97e007bd65afc/components/browse-decks/BrowseDecks.js#L66-L102

It could be like this instead:

```javascript
if (Object.keys(decks).length === 0 && uid)
  return <WelcomeRepeat createDeck={createDeck} />

return (
// ... The usual stuff
)

File: components/browse-decks/WelcomeRepeat.js (hunk 1 above)
Comment (blendaf) [reply components/browse-decks/WelcomeRepeat.js:4]: this didn't do anything that I could see

**Review by juliuscc — APPROVED** [review]

**Comment by juliuscc** [comment]
It looks great!

**Pull request merged**